### PR TITLE
Hourglass Sum 

### PR DESCRIPTION
**Pull request by Manasi2001**
I've added the complete program with proper comments, example and explanations. @rcoder23 please check out this PR with reference to issue #467.

### CODE REVIEW

File: Topic Wise Questions with Solutions/Array/Hourglass_Sum.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+0 1 0 0 0 0
+1 1 1 0 0 0
+0 0 0 0 0 0
+0 0 0 0 0 0
+0 0 0 0 0 0
+
+An hourglass in A is a subset of values with indices falling in this pattern 
+in A's graphical representation:
+a b c
+d
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Array/Hourglass_Sum.py:13]: it should be like this
a b c
&ensp; d
e f g

**Review by rcoder23 — CHANGES_REQUESTED** [review]
On running the program it give 
**unexpected EOF while parsing** error in my computer 
Recheck and correct code.

**Comment by Manasi2001** [comment]
> On running the program it give
> **unexpected EOF while parsing** error in my computer
> Recheck and correct code.

It's working absolutely fine in mine @rcoder23.
image

File: Topic Wise Questions with Solutions/Array/Hourglass_Sum.py (hunk 1 above)
Comment (Manasi2001) [reply Topic Wise Questions with Solutions/Array/Hourglass_Sum.py]: Done @rcoder23, please check now.

**Comment by rcoder23** [comment]
@Manasi2001 For test case 
0 3 0 0 0
0 1 0 0 0
1 1 1 0 0
0 0 2 4 4
0 0 0 2 4

it give output 12 but correct 11 .

**Comment by rcoder23** [comment]
Screenshot (67)

**Comment by rcoder23** [comment]
https://www.geeksforgeeks.org/maximum-sum-hour-glass-matrix/


check this.

**Comment by Manasi2001** [comment]
> @Manasi2001 For test case
> 0 3 0 0 0
> 0 1 0 0 0
> 1 1 1 0 0
> 0 0 2 4 4
> 0 0 0 2 4
> 
> it give output 12 but correct 11 .

@rcoder23 you're getting this error because you're entering a 5 X 5 matrix, whereas my code is built for a 6 X 6 array.
image
The code's link that you shared is also meant for a 5 X 5 matrix. If you'll enter any matrix with apt size, it'll give you the correct answer :)

**Comment by rcoder23** [comment]
@Manasi2001 My mistake

**Review by rcoder23 — APPROVED** [review]

**Pull request merged**